### DISCUSSION TITLE
Collaboration with Biome

### DISCUSSION THREAD

**Discussion by afonsojramos** [tailwindlabs/tailwindcss#13833] [Ideas]
Considering that the https://github.com/tailwindlabs/prettier-plugin-tailwindcss 's functionality  is being replicated in https://github.com/biomejs/biome/issues/1274, would it make sense to establish some sort of collaboration between the two projects?

**Comment by cassie2698bratt** [comment]
Hello,

Yes, it seems like a collaboration between prettier-plugin-tailwindcss and Biome could be beneficial: 
Both projects aim to sort Tailwind classes. Biome's linting with auto-fix [MaryKayInTouch](https://www.marykayin-touch.com) complements prettier-plugin-tailwindcss' formatting. Collaboration could: 

Share core sorting logic for efficiency.
Integrate prettier-plugin as a Biome linting rule.
Improve communication and development for both projects.

**Comment by stevecastaneda** [comment]
+1 Switched to biome on a large project and it's saved me a significant amount of time.  Would love to see native support for class sorting.

**Comment by adriangalilea** [comment]
Only thing preventing me to get into biome is this.

**Comment by michaelschufi** [comment]
Biome v2 beta has recently released, it includes plugin support for the linter. Maybe now would be a good time to revisit this? 

There's a Tailwind CSS Class sorting https://github.com/biomejs/biome/issues/1274 issue in the biome repo, but it hasn't had progress for a long time now.

**Reply by sebastienbarre** [reply]
My gut feeling is that since there is a rule now for Biome's linter, and that it can be auto-fixed by said linter, they are probably not going to work on the formatter doing the sorting. Which is unfortunate if you are only using Biome for the formatter, while still sticking to a differing, ESLint-compliant linter (I use OXC).
